### PR TITLE
Made 1.15 build on MacOS

### PR DESCRIPTION
**Pull request by AidanLovelace**
(In addition to not having an entry for the natives for MacOS) It didn't previously because java on Mac allows a more recently loaded classpath to override another. This was problematic here because Minecraft 1.15.2 uses LWJGL 3.2.1 while we were using LWJGL 3.2.2, though that doesn't fully explain it. I suspect it had something to do with the natives overriding and not matching the version of Minecraft natives while minecraft was still using the 3.2.1 API.

TL;DR I decremented the incremental version number to match that of Minecraft 1.15.2 of LWJGL and it fixed it. :D

### CODE REVIEW

**Comment by NotStirred** [comment]
I will test this on windows, then merge, though I dont expect any issues from the version change

**Comment by Niko-sk2x** [comment]
MC definitely uses 3.2.2, not 3.2.1. Maybe they have special version on mac. This needs to be handled differently.

**Comment by NotStirred** [comment]
Interesting, I'll change that now

**Comment by NotStirred** [comment]
Done in https://github.com/OpenCubicChunks/CubicChunks/commit/15c97831d6b2033c673b12f892ccec738304f2b5
